### PR TITLE
Added the posibility to load OCP password from a file

### PR DESCRIPTION
**Pull request by pablintino**
As a pull request owner and reviewers, I checked that:
- [x] Appropriate testing is done and actually running
- [x] Appropriate documentation exists and/or is up-to-date:

### CODE REVIEW

**Comment by pablintino** [comment]
recheck

File: ci_framework/roles/openshift_login/tasks/login.yml [hunk 1]
```diff
@@ -14,6 +14,31 @@
 # License for the specific language governing permissions and limitations
 # under the License.
 
+- name: Fetch user password from file in case is not provided by a variable
+  when:
+    - cifmw_openshift_login_password is not defined
```
Comment (dasm) [inline ci_framework/roles/openshift_login/tasks/login.yml:19]: Hmm... I just spent few minutes trying to understand how it is working, when I finally noticed the difference between
`cifmw_openshift_login_password != cifmw_openshift_login_password_file` and 
`cifmw_openshift_password != cifmw_openshift_password_file`

**Review by dasm — APPROVED** [review]
/lgtm

File: ci_framework/roles/openshift_login/tasks/login.yml (hunk 1 above)
Comment (dasm) [reply ci_framework/roles/openshift_login/tasks/login.yml:19]: I'm not sure how to make it more readable :(

**Comment by dasm** [comment]
```
TASK [Create the kubeadmin password file] **************************************
fatal: [instance]: FAILED! => changed=false
  checksum: f7c3bc1d808e04732adf679965ccc34ca7ae3441
  msg: Destination directory /home/zuul/ci-framework-data/.kube does not exist
```

**Review by raukadah — APPROVED** [review]
/approve

**Comment by psathyan** [comment]
/lgtm

Thanks for adding support for password_file. It's a nice addition

**Pull request merged**